README.md

```diff
@@ -22,7 +23,21 @@ To setup the dApp do this:
 To run the sample dApp follow these steps:
 
 1. Edit the sample dApp config file located in `.env` and add the contract addresses from the deployment executed in step 2 from Running a RelayServer. By default the project imports `.env` on the `npm start`. Look for these lines and modify them accordingly:
-
+```
+REACT_APP_CONTRACTS_RELAY_HUB=0xE0825f57Dd05Ef62FF731c27222A86E104CC4Cad
+REACT_APP_CONTRACTS_DEPLOY_VERIFIER=0x1eD614cd3443EFd9c70F04b6d777aed947A4b0c4
+REACT_APP_CONTRACTS_RELAY_VERIFIER=0x5159345aaB821172e795d56274D0f5FDFdC6aBD9
+REACT_APP_CONTRACTS_SMART_WALLET_FACTORY=0x03F23ae1917722d5A27a2Ea0Bcc98725a2a2a49a
+REACT_APP_CONTRACTS_SMART_WALLET=0x73ec81da0C72DD112e06c09A6ec03B5544d26F05
+REACT_APP_CONTRACTS_RIF_TOKEN=0x726ECC75d5D51356AA4d0a5B648790cC345985ED
+REACT_APP_CONTRACTS_TEST_RECIPIENT=0xCd5805d60Bbf9Afe69a394c2BDa10F6Dae2c39AF
+REACT_APP_CONTRACTS_RELAY_WORKER=0x3d67f029f778a088904f12d030933967d220faa3
+
+REACT_APP_ENVELOPING_CHAIN_ID=33
+REACT_APP_ENVELOPING_GAS_PRICE_FACTOR_PERCENT=0
+REACT_APP_ENVELOPING_RELAY_LOOKUP_WINDOW_BLOCKS=1e5
```

Comment:
could you please explain what these variables mean?

Reply:
@wilsoniovlabs sorry, i don't understand what you mean. could you please explain this in another way? thanks

Reply:
@wilsoniovlabs has agreed to create and work on an issue to document these variables in the SDK documentation/integration manual